### ISSUE TITLE
Will you come to dinner on Tuesday, April 16th?

### ISSUE THREAD

**Issue by sctyner** [uncoast-unconf/uu-2019#20]
Hi all, 

If you **can** come to dinner at [St Kilda](https://goo.gl/maps/LvUNCNw9sPB2) starting at 5:30pm on **Tuesday, April 16th,** give this issue a quick thumbs up 👍  (click the icon that looks like "+😄" in the top right corner of this issue). 

If you **cannot** come to dinner, give this issue a thumbs down 👎 .

If you are **unsure** give this issue a "confused face" 😕 .

I need a headcount for the restaurant by Sunday, April 14th! 

Thanks,
Sam

PS - Each person will pay for themselves. This is not funded by the Unconf. It's just an optional social gathering post-unconf!

**Comment by wendtke** [comment]
@sctyner 
Will there be group dinners on Sunday and/or Monday as well?

**Comment by sctyner** [comment]
@wendtke Yes. On Sunday we'll go to Jethro's BBQ and on Monday we will have dinner catered in to the venue. Both of these will be provided. See the full schedule [here](https://github.com/uncoast-unconf/uu-2019/wiki/Schedule)!

**Comment by wendtke** [comment]
Thanks, @sctyner. I hadn't checked the schedule for updates recently.
